cache_descriptor.go

```diff
@@ -3,11 +3,24 @@ package main
 
 import (
 	"crypto/md5"
+	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+// ChangeIndicator ...
+type ChangeIndicator string
+
+const (
+	// MD5 ...
+	MD5 = ChangeIndicator("file-mod-time")
```

Comment:
The 
`MD5 = ChangeIndicator("file-mod-time")` should be 
`ChangeIndicator("file-content-hash")` 

and the
`MODTIME = ChangeIndicator("file-content-hash")` should be 
` ChangeIndicator("file-mod-time")` instead.

They are swapped.

Reply:
@BirmacherAkos huge thanks for this catch!